employee-rostering-frontend/src/store/tenant/operations.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Tenant from 'domain/Tenant';
+import { ThunkCommandFactory } from '../types';
+import * as actions from './actions';
+import { ChangeTenantAction, RefreshTenantListAction } from './types';
+import {skillOperations} from 'store/skill';
+import { ThunkDispatch } from 'redux-thunk';
+import { Action } from 'redux';
+
+function refreshData(dispatch : ThunkDispatch<any,any,Action<any>>) : void {
+  dispatch(skillOperations.refreshSkillList());
+  skillOperations.refreshSkillList();
```

Comment:
This has no effect. `refreshSkillList()` is a "thunk" (is a pure function that returns another function). If you don't call the returned function, nothing happens. And of course, it's not even what you want because you already dispatched the operation on the previous line.